controllers/glanceapi_controller.go

```diff
@@ -809,6 +810,16 @@ func (r *GlanceAPIReconciler) generateServiceConfig(
 		return err
 	}
 
+	databaseAccount, err := mariadbv1.GetAccount(ctx, h, instance.Spec.DatabaseAccount, instance.Namespace)
```

Comment:
probably should make a helper function to make this more succinct.

I've looked at how we get things like the database name and the database hostname for generating config, to see if we can consolidate, and I dont think so.  the operators all consider the database name to be a fixed constant in their own code, so that's where that comes from, no need to look up a MariaDBDatabase for that part.    The hostname comes from up front when they create the Database facade, but as we see here, Glance is taking that hostname and putting it into the *spec* of a glanceapi instance, so that's a totally custom thing.   otherwise it seems standard right now that as controllers use Database{} to generate MariaDBDatabase, they are grabbing the hostname right there and putting it into their `instance.Status`.     Then the `Database{}` facade isn't called upon at template construction time.

There's also talk about getting a my.cnf file shuttled through this process.   But for now I think keeping "getaccount/secret" as it's own thing as we have here (I'll add a facade to get both at once) is the way forward for the moment, with possible additions of my.cnf later on.